teuthology/task/install/__init__.py

```diff
@@ -535,6 +535,7 @@ def task(ctx, config):
     if overrides:
         install_overrides = overrides.get('install', {})
         teuthology.deep_merge(config, install_overrides.get(project, {}))
+    log.debug('ROCKY_WAS_HERE')
     log.debug('config %s' % config)
```

Comment:
@kshtsk The immediate challenge I am facing is how to get the `--test-repo` value(s) to show up in the install.py config here. Once we get that, then it's just a matter of `zypper --non-interactive --no-gpg-checks addrepo` on those values, followed by `zypper --non-interactive --no-gpg-checks refresh`. Pretty trivial.